core/build.gradle

```diff
@@ -18,6 +18,18 @@ repositories {
     jcenter()
 }
 
+// Write the plugin's classpath to a file to share with the tests
+task createClasspathManifest {
```

Comment:
Hmm.. i really don't like that we have a extra task just for testing...

Reply:
still cannot find the way to add classpath without this task.

should we use `java-gradle-plugin` and try [this](https://docs.gradle.org/current/userguide/test_kit.html#sub:test-kit-automatic-classpath-injection)?

Reply:
Yeah, give it a try 👍